Gradle toolchain cannot automatically download JDK 8 for aarch64 on macOS

I am using the current main branch on commit 4a87980a

## Steps to reproduce

Take an Apple Silicon Device and try `./gradlew clean assemble`, it will produce the following output:
```
╰$ ./gradlew clean assemble        

Welcome to Gradle 7.5.1!

Here are the highlights of this release:
 - Support for Java 18
 - Support for building with Groovy 4
 - Much more responsive continuous builds
 - Improved diagnostics for dependency resolution

For more details see https://docs.gradle.org/7.5.1/release-notes.html

Type-safe project accessors is an incubating feature.

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':junit-platform-console-standalone:jar'.
> Could not resolve all dependencies for configuration ':junit-platform-console-standalone:shadowed'.
   > Failed to query the value of task ':junit-vintage-engine:compileTestFixturesGroovy' property 'javaLauncher'.
      > Unable to download toolchain matching these requirements: {languageVersion=8, vendor=any, implementation=vendor-specific}
         > Unable to download toolchain. This might indicate that the combination (version, architecture, release/early access, ...) for the requested JDK is not available.
            > Could not read 'https://api.adoptium.net/v3/binary/latest/8/ga/mac/aarch64/jdk/hotspot/normal/eclipse' as it does not exist.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
12 actionable tasks: 12 up-to-date

A build scan was not published as you have not authenticated with server 'ge.junit.org'.
For more information, please see https://gradle.com/help/gradle-authenticating-with-gradle-enterprise.
```

Seems like spotbugs is trying to get aarch64 version of jdk8, but this is not available. Would it be possible to set the platform to amd64?

## Context

 - Used versions (Jupiter/Vintage/Platform): all
 - Build Tool/IDE: IntelliJ IDEA and Mac Terminal

--- a/README.md
+++ b/README.md
@@ -66,8 +66,8 @@ You need [JDK 17] to build JUnit 5. [Gradle toolchains] are used to detect and
 potentially download additional JDKs for compilation and test execution.
 
 Note: If you're working on a machine with an Apple Silicon processor, you'll need to
-download and install a JDK 8 distribution manually (or via [SDKMAN!] or [asdf]),
-e.g. the one from [Zulu] or [Liberica].
+download and install a JDK 8 distribution manually (or via [SDKMAN!] or [asdf]) --
+for example, one from [Zulu] or [Liberica].
 
 All modules can be _built_ with the [Gradle Wrapper] using the following command.
 
@@ -112,4 +112,4 @@ See also <https://repo1.maven.org/maven2/org/junit/> for releases and
 [SDKMAN!]: https://sdkman.io/jdks
 [StackOverflow]: https://stackoverflow.com/questions/tagged/junit5
 [User Guide]: https://junit.org/junit5/docs/current/user-guide/
-[Zulu]: https://www.azul.com/downloads/?version=java-8-lts&os=macos&architecture=arm-64-bit&package=jdk
+[Zulu]: https://www.azul.com/downloads/?version=java-8-lts&os=macos&architecture=arm-64-bit&package=jdk#download-openjdk

--- a/README.md
+++ b/README.md
@@ -65,6 +65,10 @@ task outputs from previous CI builds.
 You need [JDK 17] to build JUnit 5. [Gradle toolchains] are used to detect and
 potentially download additional JDKs for compilation and test execution.
 
+Note: If you're working on a machine with an Apple Silicon processor, you'll need to
+download and install a JDK 8 distribution manually (or via [SDKMAN!] or [asdf]),
+e.g. the one from [Zulu] or [Liberica].
+
 All modules can be _built_ with the [Gradle Wrapper] using the following command.
 
 `gradlew clean assemble`
@@ -92,6 +96,7 @@ See also <https://repo1.maven.org/maven2/org/junit/> for releases and
 <https://oss.sonatype.org/content/repositories/snapshots/org/junit/> for snapshots.
 
 
+[asdf]: https://asdf-vm.com/
 [Codecov]: https://codecov.io/gh/junit-team/junit5
 [CONTRIBUTING.md]: https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md
 [Dependency Metadata]: https://junit.org/junit5/docs/current/user-guide/#dependency-metadata
@@ -101,7 +106,10 @@ See also <https://repo1.maven.org/maven2/org/junit/> for releases and
 [JaCoCo]: https://www.eclemma.org/jacoco/
 [Javadoc]: https://junit.org/junit5/docs/current/api/
 [JDK 17]: https://adoptium.net/archive.html?variant=openjdk17&jvmVariant=hotspot
+[Liberica]: https://bell-sw.com/pages/downloads/
 [Release Notes]: https://junit.org/junit5/docs/current/release-notes/
 [Samples]: https://github.com/junit-team/junit5-samples
+[SDKMAN!]: https://sdkman.io/jdks
 [StackOverflow]: https://stackoverflow.com/questions/tagged/junit5
 [User Guide]: https://junit.org/junit5/docs/current/user-guide/
+[Zulu]: https://www.azul.com/downloads/?version=java-8-lts&os=macos&architecture=arm-64-bit&package=jdk

